riscv/runtime/src/coprocessors.rs

```diff
@@ -5,8 +5,16 @@
 // during the reachability analysis.
 extern "C" {
     fn poseidon_coprocessor(a: u32, b: u32) -> u32;
+    fn poseidon_gl_coprocessor(a: PoseidonGLU256, b: PoseidonGLU256, c: PoseidonGLU256) -> PoseidonGLU256;
 }
 
+#[repr(C)]
+pub struct PoseidonGLU256(pub u64, pub u64, pub u64, pub u64);
```

Comment:
We needed the struct and the repr(C) directive because otherwise the compiler complains this is FFI unsafe

Reply:
right, hence my suggestion to use `[u64]` instead, so that we are less prone to this kind of ABI restrictions.